src/Microsoft.Extensions.Configuration.EnvironmentVariables/EnvironmentVariablesConfigurationSource.cs

```diff
@@ -1,17 +1,23 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Linq;
-
 namespace Microsoft.Extensions.Configuration.EnvironmentVariables
 {
+    /// <summary>
+    /// A environment variable based <see cref="IConfigurationSource"/>.
```

Comment:
In https://github.com/aspnet/Configuration/pull/461/files/100a4a6ad7a4b371b3068e623da2bb498936851b#diff-1f0a2c8c46328cea7cc9f9473bab8bb5R9 you used a completely different approach: `Settings for the <see cref="CommandLineConfigurationProvider"/>`. Neither approach seems to add much useful information but it would be good to at least be consistent.